### ISSUE TITLE
pnpm output doesn't look right in git bash shell for Windows

### ISSUE THREAD

**Issue by kaiyoma** [pnpm/pnpm#3376]
### Expected behavior:

I'm not sure actually, but according to the author in a Discord chat, this output (below) isn't right. There's a new line for every update and the `readPackage` hook text is indented too much. I see this behavior when doing a `rush update` but also when running pnpm directly.

### Actual behavior:

![pnpm-output](https://user-images.githubusercontent.com/10719780/115595184-790d4a00-a28b-11eb-9f03-e30ca2996bad.png)

### Additional information:

 - `node -v` prints: 14.15.0
 - Windows, macOS, or Linux?: Windows 10

**Comment by zkochan** [comment]
You said it works with npm/yarn.

I get similar output with npm/yarn: no colors, each print is on a new line

![image](https://user-images.githubusercontent.com/1927579/115613326-f0c38f00-a2f4-11eb-8ba8-2aa309896b5e.png)

**Comment by kaiyoma** [comment]
Your yarn output looks correct to me, meaning that the progress bar was rendered correctly. In your screenshot, the pnpm progress bar looks correct (always on the same line), which is also how yarn works.

**Comment by zkochan** [comment]
No, Yarn had no progress bar. It was just that text. And pnpm output was like this:

![Uploading image.png…]()

**Comment by kaiyoma** [comment]
Okay, so you've repro'd the pnpm output issue then?

yarn should show a progress bar for fetching and linking. I think the issue is that you're not actually installing anything? Try adding some packages and then you'll see the progress bars:

![image](https://user-images.githubusercontent.com/10719780/115635569-dd96cc00-a2c0-11eb-86e7-659607b08b9a.png)

![image](https://user-images.githubusercontent.com/10719780/115635589-e7b8ca80-a2c0-11eb-8160-efced044fc71.png)

**Comment by zkochan** [comment]
No, I add lots of packages. No progress bar.

**Comment by zkochan** [comment]
Closing for lack of a reproduction on a supported version.

A git bash rendering report against pnpm 5.18 through Rush. The default reporter falls back to append-only output when stdout is not a terminal, and `--reporter=append-only` forces it.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
